src/controllers/login.controller.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { Test } from '@nestjs/testing';
+import { loginDTOMock } from '../dtos/__mocks__/login.dto';
+import { JwtResultMock } from '../models/__mocks__/jwt-result.model';
```

Comment:
I suggest changing to:
```suggestion
import { jwtResultMock } from '../models/__mocks__/jwt-result.model';
```

Reply:
Resolved on a1150ef6e27ae4d69f22689086f22476fce1ff8c.